### DISCUSSION TITLE
What to do after the new update

### DISCUSSION THREAD

**Discussion by ZuspOfReal** [ShimmyMySherbet/ShimmysAdminTools#32] [Q&A]
After a new update, the plugins on the server disappeared. What can be done?
Kits work, /fixv/refuel and others do not work.

**Comment by neyker001-hub** [comment]
same for me i cant load the plugin  it say "An error occured while executing rocket [load, ShimmysAdminTools]: System.NullReferenceException: Object reference not set to an instance of an object"

**Reply by ShimmyMySherbet** [reply]
When did you start having issues, and are you using the latest version of the plugin?
Also heads up, it's best to restart the server when having plugin issues, using the /load command can break a lot of plugins.

**Reply by neyker001-hub** [reply]
no no i just unisntall the plugin because it dont load when the server start i have this issue when i update to the last rocket module . sorry for my bad english
